ember.md

```diff
@@ -47,6 +47,17 @@ lastName: attr('string'),
 fullName: belongsTo('person')
 ```
 
+- 1.3 Default Unassigned Property
```

Comment:
Keep consistent with existing format.
"Default Unassigned Property" should be bold and "When declaring an unassigned..." should start a new line.